test/components/tree-view.spec.tsx

```diff
@@ -47,6 +48,45 @@ const treeData: TreeItemData[] = [
     }
 ];
 
+const newTreeData = JSON.parse(JSON.stringify(treeData));
+newTreeData[0].children![2].children!.push({label: 'Kaiserschmarrn'});
+
+export interface TreeViewWrapperState {
+    treeData: object[];
+}
+
+export class TreeViewWrapper extends React.Component<{}, TreeViewWrapperState> {
+
+    constructor() {
```

Comment:
You don't need to call the constructor here, just use: 
public state = {treeData};

Reply:
done